### PR TITLE
Raise exception on empty filePath

### PR DESCRIPTION
**Pull request by simiaosimis**
Raise a ConfigurationFileError when start parser gets a empty filePath.

This closes issue https://github.com/Kuniri/kuniri/issues/88

### CODE REVIEW

File: lib/kuniri/parser/parser.rb [hunk 1]
```diff
@@ -23,7 +23,9 @@ def initialize(pFilesPath, pLanguage = "ruby")
 
         # Start parse in the project.
         def start_parser
-          # TODO: handle the case of filePAth is empty
+          # TODO: handle the case of filePath is empty
+          raise Error::ConfigurationFileError if(@filesPath.empty?)
+
```
Comment (rodrigosiqueira) [inline lib/kuniri/parser/parser.rb:30]: Add a message in log, if @filesPath.empty? is true.

File: lib/kuniri/parser/parser.rb [hunk 2]
```diff
@@ -23,7 +23,9 @@ def initialize(pFilesPath, pLanguage = "ruby")
 
         # Start parse in the project.
         def start_parser
-          # TODO: handle the case of filePAth is empty
+          # TODO: handle the case of filePath is empty
+          raise Error::ConfigurationFileError if(@filesPath.empty?)
```
Comment (rodrigosiqueira) [inline lib/kuniri/parser/parser.rb]: Add space after "if"

File: lib/kuniri/parser/parser.rb (hunk 1 above)
Comment (simiaosimis) [inline lib/kuniri/parser/parser.rb:30]: Do you mean a message log instead the exception or should I do both?

File: lib/kuniri/parser/parser.rb (hunk 2 above)
Comment (rodrigosiqueira) [inline lib/kuniri/parser/parser.rb]: if (@filesPath.empty?)

File: lib/kuniri/parser/parser.rb (hunk 1 above)
Comment (rodrigosiqueira) [inline lib/kuniri/parser/parser.rb:30]: Both :)

**Comment by rodrigosiqueira** [comment]
Merged.

**Pull request closed without merging**